src/adagrams.js

```diff
@@ -1,15 +1,145 @@
+//helper functions
+
+// const randomInt = function (max) {
+//   return Math.floor(Math.random() * max);
+// };
+const randomInt = (max) => Math.floor(Math.random() * max);
+
+const removeLetter = function (arr, target) {
+  let i = arr.indexOf(target);
+  if (i > -1) {
+    arr.splice(i, 1);
+  }
+  return arr;
+};
+
+const uppercaseArray = (str) => {
+  let string = str.toUpperCase();
+  return Array.from(string);
+};
+
+//main functions
+
+//Wave 1
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const lettersObject = {
```

Comment:
This can be declared outside of the function so that it doesn't have to be recreated each time the function is called.